commands/verify/verify.go

```diff
@@ -101,9 +101,16 @@ func DoVerifyAccordingConfig() error {
 		return err
 	}
 
+	failFast := e2eConfig.Verify.FailFast
+
 	for idx, v := range e2eConfig.Verify.Cases {
 		if v.GetExpected() == "" {
-			return fmt.Errorf("the expected data file for case[%v] is not specified", idx)
+			errMsg := fmt.Sprintf("the expected data file for case[%v] is not specified", idx)
+			if !failFast {
+				logger.Log.Warnf(errMsg)
+				continue
+			}
+			return fmt.Errorf("%s",errMsg)
```

Comment:
```suggestion
			return errors.New(errMsg)
```

You should use the `errors.New` to create the error. Don't forget to import the package.

Reply:
Thanks  for your explanation!

Reply:
Please add the "errors" package import in this file.

Reply:
OK!

Reply:
Please fix the code style of the import package. You can use the `make lint` to check it locally.

Reply:
OK ! Thanks for your patient explanation !